docs/dev/convert_layout.rst

```diff
@@ -116,6 +117,7 @@ These steps happen for each operator in sequence, where ConvertLayout pass keeps
         data_layout = attrs['data_layout']
         kernel_layout = attrs['kernel_layout']
         data, weight = inputs
+        desired_layout = str(desired_layouts[0])
         assert desired_layout == 'NCHW', \
                 "Currently only transformation to NCHW layout is supported."
         if desired_layout == 'NCHW':
```

Comment:
This checking seems not necessary because Line 121 already has an assertion. Either remove this branch or move the assertion to the else branch.